demo/hacktoberfest-custom-themes/index.html

```diff
@@ -72,11 +72,10 @@
     <div class="column col-11 col-mx-auto">
         <div class="column col-10 col-mx-auto">
             <h2>Hacktoberfest diagram</h2>
-            <p>You can use this diagram to promote your project in order to have contributions during Hacktoberfest:
-            <ul>
-                <li>Enter your project name</li>
-                <li>Take a screenshot of your favorite diagram theme</li>
-                <li>Advertise about your project with the diagram (Don't forget to add a link to this page)</li>
+            <p class="mb-0">You can use this diagram to promote your project in order to have contributions during Hacktoberfest:
+            <ul class="mt-0">
+                <li>Make it your own: change the project name to yours and choose your favorite theme</li>
+                <li>Share: take a screenshot and advertise about your project (don't forget to add a link to this page)</li>
```

Comment:
**suggestion:** alternate text
don't forget to add a link to this page --> don't forget to credit this page

Reply:
I prefer the suggestion 😊

Reply:
done in d0676390e71de11857a7ad4806782c44a46e2fa5